roles/ceph-common/tasks/main.yml

```diff
@@ -26,15 +26,15 @@
   when: ansible_os_family == 'Debian'
 
 - name: check for a ceph socket
-  shell: "stat /var/run/ceph/*.asok > /dev/null 2>&1"
+  stat: path=/var/run/ceph/*.asok
```

Comment:
I'm not sure about this one, AFAIR stat doesn't support * 
Did you get a chance to validate this?